resources/js/admin.js

```diff
@@ -84,3 +83,47 @@ document.addEventListener('DOMContentLoaded', function () {
         }
     })
 })
+
+jQuery(document).ready(function ($) {
```

Comment:
@Muniraj132, we've been using vanilla JS in our projects. Please convert this to Vanilla JS to keep continuity. It's fine to use jQuery if necessary (like if you're interacting with a jQuery library), but please stick with vanilla JS when possible.

Reply:
@incraigulous  I’ve converted the script from jQuery to Vanilla JS for consistency. Please review and approve the PR at your convenience.